sormas-api/src/main/java/de/symeda/sormas/api/common/DeleteReason.java

```diff
@@ -0,0 +1,18 @@
+package de.symeda.sormas.api.common;
+
+import de.symeda.sormas.api.i18n.I18nProperties;
+
+public enum DeleteReason {
```

Comment:
```suggestion
public enum DeletionReason {
```

Please also rename all fields accordingly

Reply:
changed